README.md

```diff
@@ -1 +1,21 @@
 ### ReadMe Test
+def mod_func(a,b):
+  return a%b
+
+def addition_func(a,b):
+  return a+b
+
+def mult_func(a,b):
+  c=a*b
+  return c
+
+
+a=51
+b=11
+sum = addition_func(a,b)
+mod = mod_func(b,a)
+mul = mult_func(a,b)
+
+print(sum)
```

Comment:
This line prints the value of the variable 'sum'. No issues or improvements needed.